src/types.ts

```diff
@@ -390,16 +396,27 @@ export interface JSONRPCPayload {
 }
 
 /*
+ * pollingIntervalMs: How often to poll the Ethereum node for new events.
+ * numConfirmations: How many confirmed blocks deep you wish to listen for events at.
+ */
+export interface OrderStateWatcherConfig {
+    pollingIntervalMs?: number;
```

Comment:
should these also follow the "ifExists" convention

Reply:
No since it's a public config variable and the user will only define it if they want it to exist.